settings.py

```diff
@@ -1928,7 +1924,6 @@ def _pkce_required(client_id):
 # Adding a domain will not automatically index that domain's existing forms
 ES_XFORM_FULL_INDEX_DOMAINS = [
     'commtrack-public-demo',
```

Comment:
so close to being able to remove these ES indices!

Reply:
I was wondering about that. Is that domain still being used? I'll check on it.